Marlin/src/module/endstops.cpp

```diff
@@ -672,14 +672,17 @@ void Endstops::update() {
   }while(0)
 
   #if ENABLED(G38_PROBE_TARGET) && PIN_EXISTS(Z_MIN_PROBE) && !(CORE_IS_XY || CORE_IS_XZ)
+    #if ENABLED(G38_PROBE_AWAY)
+      #define _G38_OPEN_STATE (G38_move > 2)
```

Comment:
@thinkyhead I think this is a bug, this should be a 3

Reply:
Thanks! I usually double-check my late-night coding the next day, but I was about ready to just merge and forget!